### PR TITLE
Add Eloquent JavaScript (1er, 2do, 3ra, 4ta Edicion)

### PR DESCRIPTION
**Pull request by GNUXDAR**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

### Description
A link is added to a personal public repository where the 4 editions of the book Eloquent JavaScript are stored.

### Why is this valuable (or not)?
It is valuable because it contains the 4 editions released so far, with links to the HTML version and PDFs.

### How do we know it's really free?
It is free because it is a book that was created and previously added to this repository, translated by the JavaScript community.

### For book lists, is it a book? For course lists, is it a course? etc.
These are all the editions of the Eloquent JavaScript books released to date.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
seems to me the proper 4th edition link is https://www.eloquentjavascript.es/ 
the 3rd edition is already there
the 2nd and 1st editions are the english ones

**Comment by GNUXDAR** [comment]
Yes, the first and second editions are in English. Should I remove them or just keep the links to the fourth edition in Spanish?

**Comment by eshellman** [comment]
remove the link to your repo and add the direct link for the 4th edition

**Comment by GNUXDAR** [comment]
I have updated the link to the Spanish version of the fourth edition and added a link to the PDF.

File: books/free-programming-books-es.md [hunk 1]
```diff
@@ -205,7 +205,8 @@
 ### JavaScript
 
 * [El Tutorial de JavaScript Moderno](https://es.javascript.info) - Ilya Kantor, Elizabeth Portilla, joaquinelio, Ezequiel Castellanos, et al. (HTML)
-* [Eloquent JavaScript](https://eloquentjs-es.thedojo.mx) - Marijn Haverbeke, `trl.:` Various (HTML, PDF, EPUB, MOBI)
+* [Eloquent JavaScript (3ra Edición)](https://eloquentjs-es.thedojo.mx) - Marijn Haverbeke, `trl.:` Various (HTML, PDF, EPUB, MOBI)
+* [Eloquent JavaScript (4ta Edición)](https://www.eloquentjavascript.es) - Marijn Haverbeke (HTML) [(PDF)](https://github.com/GNUXDAR/js-de-0-a-experto/blob/master/Docs/EbookEloquent/Eloquent_JavaScript_4ta_ES.pdf)
```
Comment (eshellman) [inline books/free-programming-books-es.md:209]:
```suggestion
* [Eloquent JavaScript (4ta Edición)](https://www.eloquentjavascript.es) - Marijn Haverbeke (HTML) (PDF)
```
There is already a link to the PDF on the eloquentjavascript.es page. extra links create a mainenance burden for us.

Comment (GNUXDAR) [reply books/free-programming-books-es.md]: Ok, i go update commit

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Comment by GNUXDAR** [comment]
You're welcome, I have the fork and will probably make more frequent PRs. I'm about to create a web development course, which will most likely be uploaded here. Thanks for everything, Mr. Eric.

**Pull request merged**